docs/UserGuide.md

```diff
@@ -64,31 +70,48 @@ EduManage is a **desktop app for managing contacts, optimized for use via a Comm
+|   3   |       Index       |                    `INDEX`                     | Should be a positive integer equal to or less than the size of the current displayed student list                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+|   4   |    Lesson Time    |                `lt/LESSON_TIME`                | Should be in format `DAY-STARTTIME-ENDTIME`, where:<ul><li>`DAY` is in:<br/>[MON, TUE, WED, THU, FRI, SAT, SUN]</li><li>`STARTTIME` and `ENDTIME` are in the format `HH:MM`, `HH` must be between 0 and 23 inclusive, `MM` must be between 0 and 59 inclusive</li><li>`ENDTIME` must be later than `STARTTIME`</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|   5   |       Level       |                   `l/LEVEL`                    | Should be in the format `YEAR TRACK`, where:<ul><li>`YEAR` is one of:<br/>[S1, S2, S3, S4, S5]</li><li>`TRACK` is one of:<br/>[EXPRESS, NA, NT, IP]<br/>with the exception of S5 which is only allowed to have the `TRACK` NA</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
same for this, no need to use line break except for the with the exception of ...